config/routes.rb

```diff
@@ -3,12 +3,16 @@
 
   get '/netflix_shows', to: "homes#index"
   get '/netflix_shows/:id', to: "homes#index"
+  get '/netflix_shows/:id/add_review', to: "homes#index"
   
   devise_for :users
 
   namespace :api do
     namespace :v1 do
-      resources :netflix_shows, only: [:index, :show]
+      resources :netflix_shows, only: [:index, :show] do
+        resources :reviews, only: [:create, :new]
+      end
+
```

Comment:
good, though you probably wont need `:new` since this will be on the show page, not on its on individual route